app/models/service_template.rb

```diff
@@ -74,6 +79,7 @@ class ServiceTemplate < ApplicationRecord
   scope :without_service_template_catalog_id,       ->         { where(:service_template_catalog_id => nil) }
   scope :with_existent_service_template_catalog_id, ->         { where.not(:service_template_catalog_id => nil) }
   scope :displayed,                                 ->         { where(:display => true) }
+  scope :public_service_templates,                  ->         { where.not(:service_type => 'internal') }
```

Comment:
Would just `public` make more sense?

Reply:
`public` is reserved keyword. I would rather not to cause confusion even if it is allowed here.

Reply:
:+1: